js/print.js

```diff
@@ -0,0 +1,26 @@
+var jobs = [];
+
+function job (shit) {
```

Comment:
While I understand, this is not the most professional of variable names. You should change it before you show this off to people that aren't me.